mkdocs.yml

```diff
@@ -15,6 +15,8 @@ nav:
     - Further information: guide/further-information.md
   - Design:
     - Index: design/index.md
+  - ML Toolkit:
+    - Index: ml-toolkit/index.md
```

Comment:
I think it would be nice if the other parts are listed here as well so they are listed in the side bar, like for the user guide: 
![image](https://user-images.githubusercontent.com/23457217/209086321-56f68680-cd99-4267-bd16-1e11182e2871.png)

Otherwise, if I am not mistaken, it will look like for home: 
![image](https://user-images.githubusercontent.com/23457217/209086442-aa4015c1-821d-45c6-aabc-55dd04ba10e7.png)